nosql/1.0/_index.md

```diff
@@ -46,12 +46,12 @@ template.delete(Car.class, 1L);
 
 
 * [Jakarta NoSQL Release Plan](https://projects.eclipse.org/projects/ee4j.nosql/governance)
-* [Jakarta NoSQL 1.0 Specification Document](./nosql-1.0.0-M1.pdf) (PDF)
-* [Jakarta NoSQL 1.0 Specification Document](./nosql-1.0.0-M1.html) (HTML)
+* [Jakarta NoSQL 1.0 Specification Document](./nosql-1.0.0.pdf) (PDF)
+* [Jakarta NoSQL 1.0 Specification Document](./nosql-1.0.0.html) (HTML)
 * [Jakarta NoSQL 1.0 Specification Javadoc](./apidocs)
 * Maven coordinates
-  * [jakarta-nosql-api:jakarta.nosql:jar:1.0.0-M1](https://repo1.maven.org/maven2/jakarta/nosql/jakarta.nosql-api/1.0.0-M1/)
-
+  * [jakarta-nosql-api:jakarta.nosql:jar:1.0.0](https://repo1.maven.org/maven2/jakarta/nosql/jakarta.nosql-api/1.0.0)
```

Comment:
Could the displayed
- groupId be changed to `jakarta.nosql` (from `jakarta-nosql-api`)
- artifactId to `jakarta.nosql-api` (from `jakarta.nosql`)?

Reply:
Done.

Reply:
> Done.

Can it be pushed then?

Reply:
Could you check it?

Reply:
It's fine now, thank you.